src/openeo_pg_parser/utils.py

```diff
@@ -75,9 +75,13 @@ def load_processes(src):
             filepaths = glob.glob(os.path.join(src, "*.json"))
             process_list = [load_json_file(filepath) for filepath in filepaths]
         elif isinstance(src, str) and url_is_valid(src):
+            # Is it the URL of a JSON file or a /processes endpoint?
             r = requests.get(url=src)
             data = r.json()
-            process_list = data['processes']
+            if src.split('.')[-1].lower() == 'json':
```

Comment:
I think this assumption may not always be true. I guess it's better to simply check whether you get an object with a "processes" property or whether you get an array and then handle it accordingly.